projects/backend/controllers/render.ts

```diff
@@ -305,20 +336,25 @@ export const renderItemController = async (req: Request, res: Response) => {
     }
 
     // find the corresponding furniture for the article so we can apply the fronts overrides
-    const idFurniturePair = front.collections
+    const groupIdToFurnitureAndTheme = front.collections
```

Comment:
Is this function just a repeat of line 282? If so could we extract it into something reusable?